docs/how_to_guide/adding_a_coalignment_method.rst

```diff
@@ -0,0 +1,44 @@
+.. _add-a-new-coalignment-method:
```

Comment:
These should be very specific as they were before.

We want to make sure that sphinx labels like this are not generic otherwise it can cause problems.